jit/v2/JIT_v2.0.0.yaml

```diff
@@ -165,6 +292,102 @@ paths:
           headers:
             API-Version:
               $ref: '#/components/headers/API-Version'
+        '400':
+          description: |
+            In case creating a new `Estimated Timestamp` does not schema validate a `400` (Bad Request) is returned
```

Comment:
In case creating a new 'Estimated Timestamp' fails schema validation a 400 (Bad Request) is returned